### PR TITLE
Fix vpn example. Expand logs

### PR DESCRIPTION
**Pull request by denis-tingajkin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

## Motivation and Context
https://github.com/networkservicemesh/networkservicemesh/issues/1397

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: k8s/conf/vpn-gateway-nse.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
             - name: TRACER_ENABLED
               value: "true"
             - name: IP_ADDRESS
-              value: "10.60.1.0/24"
+              value: "172.16.1.0/24"
```
Comment (nickolaev) [inline k8s/conf/vpn-gateway-nse.yaml]: Why this change?

Comment (denis-tingajkin) [reply k8s/conf/vpn-gateway-nse.yaml]: To exclude case with IP intersections on clouds. Because by logs which I saw looks like it is a root cause. Also, I've expanded logs for bash tests to get more information about the problem if it is still reproducible.

Comment (denis-tingajkin) [reply k8s/conf/vpn-gateway-nse.yaml]: Now bash tests logs save on CI in artifacts directory.
